tests/upgrade/test_existence.py

```diff
@@ -0,0 +1,139 @@
+"""Upgrade TestSuite for validating Satellite entities existence and
+associations post upgrade
+"""
+import unittest2
```

Comment:
What about to `import unittest2 as unittest`. That will make easy to convert to Python 3 later on.